core/src/main/java/me/deadlight/ezchestshop/data/Config.java

```diff
@@ -109,9 +109,9 @@ public static void loadConfig() {
 
         EzChestShop.getPlugin().reloadConfig();
         FileConfiguration config = EzChestShop.getPlugin().getConfig();
-        taxesshopplayer = config.getInt("economy.taxes-shop-player");
-        taxesplayershop = config.getInt("economy.taxes-player-shop");
-        leader = config.getString("economy.leader");
+        taxesbuy = config.getInt("economy.taxes-buy-from-shop-tax-percent");
+        taxessell = config.getInt("economy.taxes-sell-to-shop-tax-percent");
```

Comment:
You might wanna indent this yml entry (also in `config.yml`), so it's `economy.taxes.tax-percent.sell-to-shop` and `economy.taxes.tax-percent.buy-from-shop`. Easier to read then a long string and generally less to write